framework/Crypt/lib/Horde/Crypt/Smime.php

```diff
@@ -174,29 +174,30 @@ public function verify($text, $certs)
      * Extract the contents from signed S/MIME data.
      *
      * @param string $data     The signed S/MIME data.
-     * @param string $sslpath  The path to the OpenSSL binary.
      *
      * @return string  The contents embedded in the signed data.
      * @throws Horde_Crypt_Exception
      */
-    public function extractSignedContents($data, $sslpath)
+    public function extractSignedContents($data)
```

Comment:
Please keep the $sslpath parameter there for backward compatibility, but mark it as deprecated in the phpdoc.